src/Instance/InstanceMarc/InstanceMarc.css

```diff
@@ -0,0 +1,10 @@
+.instanceMarcWrapper {
+  margin-left: 20px;
```

Comment:
isn't there a constant for such paddings?

Reply:
I'm not sure, but it was before in inline styles